src/main/java/dk/kb/datahandler/util/HarvestTimeUtil.java

```diff
@@ -49,6 +62,75 @@ public static synchronized void updateDatestampForOaiTarget(OaiTargetDto oaiTarg
         updateDatestampForOaiTarget(oaiTargetNameFile, datestamp);        
     }
     
+    /**
+     * Validate day format is of form at yyyy-MM-dd.
+     * Example: 2024-02-09
+     * 
+     * @param day  Validate is of correct format
+     */
+    public static boolean validateDayFormat(String day) {
+    	SimpleDateFormat simpleDateFormat = new SimpleDateFormat(dayPattern,Locale.getDefault());
```

Comment:
Never use `Locale.getDefault()`, unless it is for the creation of system locale dependent Strings (to my knowledge we don't use that anywhere). `Locale.ROOT` is a nice go-to.

The same goes for the other uses of `Locale.getDefault()` in the code.